bin/install-wp-tests.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
```

Comment:
Can you add a comment to state this file comes from https://github.com/wp-cli/sample-plugin/blob/c6dd999c24442be4550b085a4e29f91e92137058/bin/install-wp-tests.sh ?
So that we can keep track of it.